ncm-directoryservices/src/main/pan/components/directoryservices/schema.pan

```diff
@@ -24,5 +24,7 @@ type component_directoryservices = {
         "ldapv3" ? directoryservices_ldap_entry{}
 };
 
-bind "/software/components/directoryservices" = component_directoryservices;
-
+bind "/software/components/directoryservices" = component_directoryservices with {
```

Comment:
any reason why this is done at the bind, not at the type itself? (not that i care 😄 )